pkg/capr/planner/planner.go

```diff
@@ -127,6 +128,12 @@ type Planner struct {
 	locker                        locker.Locker
 	etcdS3Args                    s3Args
 	retrievalFunctions            InfoFunctions
+
+	// Equalities provides a custom interface for `DeepEqual`, so NodePlan's can be compared
+	// (which have a function pointer field) without attempting to compare function pointers.
+	// DeepEqual cannot compare function pointer, unless that are `nil`, so otherwise it would
+	// always return false.
+	Equalities conversion.Equalities
```

Comment:
This is the only exportable field in the Planner. Does it need to be exportable?

Reply:
Fixed.